content/SCALE/GettingStarted/FirstTimeLogin.md

```diff
@@ -185,11 +185,11 @@ The *Email* screen has options for users to set up a system email address.
 | Security | [Email encryption](https://www.fastmail.com/help/technical/ssltlsstarttls.html) type. Choices are Plain (No Encryption), SSL (Implicit TLS), or TLS (STARTTLS). |
```

Comment:
Are the choices mentioned UI provided options? if yes use bold. Also, are the ( ) enclosed descriptions part of the UI choice? If yes, bold as well. 
```suggestion
| Security | [Email encryption](https://www.fastmail.com/help/technical/ssltlsstarttls.html) type. Choices are **Plain (No Encryption)**, **SSL (Implicit TLS)**, or **TLS (STARTTLS)**. |
```